app/actions/CustomActionBuilders.scala

```diff
@@ -31,30 +31,36 @@ class CustomActionBuilders(
 
   import CustomActionBuilders._
 
+  val membersIdentityClientId = "clientId" -> "members"
+
+  val recurringIdentityClientId = "clientId" -> "recurringContributions"
+
   // Tells identity to send users back to the checkout immediately after sign-up.
   private val idSkipConfirmation: (String, String) = "skipConfirmation" -> "true"
   // Prevents the identity validation email sending users back to our checkout.
   private val idSkipValidationReturn: (String, String) = "skipValidationReturn" -> "true"
 
-  private val idMember = "clientId" -> "members"
-
-  private def idWebAppRegisterUrl(path: String): String =
-    idWebAppUrl / "register" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & idMember
+  private def idWebAppRegisterUrl(path: String, clientId: (String, String)): String =
+    idWebAppUrl / "register" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & clientId
 
-  private def newSignInFlowIdWebAppRegisterUrl(path: String): String =
-    idWebAppUrl / "signin/start" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & idMember
+  private def newSignInFlowIdWebAppRegisterUrl(path: String, clientId: (String, String)): String =
+    idWebAppUrl / "signin/start" ? ("returnUrl" -> s"$supportUrl$path") & idSkipConfirmation & idSkipValidationReturn & membersIdentityClientId
```

Comment:
Does this need to take `clientId` as a param if it's not used?

Reply:
@joelochlann @Ap0c This is supposed to be clientId, I spotted this when I was showing you earlier Jamie but I didn't push before I went to lunch, will change it now

Reply:
yes, this should be `clientId`, see my other comment